diff --git a/Makefile b/Makefile
index abef9a9..2ecd2fc 100644
--- a/Makefile
+++ b/Makefile
@@ -43,8 +43,6 @@ dist: clean
 	@rm -rf dvtm-${VERSION}
 
 install: dvtm
-	@echo stripping executable
-	@strip -s dvtm
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	@cp -f dvtm ${DESTDIR}${PREFIX}/bin
diff --git a/config.mk b/config.mk
index 05106e5..e957080 100644
--- a/config.mk
+++ b/config.mk
@@ -3,7 +3,7 @@ VERSION = 0.12
 
 # Customize below to fit your system
 
-PREFIX = /usr/local
+PREFIX = /usr
 MANPREFIX = ${PREFIX}/share/man
 
 INCS = -I.
@@ -13,7 +13,7 @@ LIBS = -lc -lutil -lncursesw
 # AIX
 #LIBS = -lc -lncursesw
 # Cygwin
-#INCS += -I/usr/include/ncurses
+INCS += -I/usr/include/ncursesw
 
 CFLAGS += -std=c99 -Os ${INCS} -DVERSION=\"${VERSION}\" -DNDEBUG
 LDFLAGS += ${LIBS}
